src/main/resources/application.yaml

```diff
@@ -127,6 +127,14 @@ security:
     caseworker-ia-admofficer:
       - "listCase"
       - "recordAttendeesAndDuration"
+    caseworker-ia-homeofficeapc:
+      - "uploadRespondentEvidence"
+    caseworker-ia-homeofficelart:
+      - "addAppealResponse"
+    caseworker-ia-homeofficepou:
+      - "sendDirection"
+    caseworker-ia-respondentofficer:
+      - "sendDirection"
```

Comment:
My understanding is respondent office should be able to do everything that other homeoffice roles could do. Eg: addAppealResponse and uploadRespondentEvidence. Could you pleae clarify?

Reply:
I think you're correct, thanks @jkumar-moj